ksql-engine/src/main/java/io/confluent/ksql/planner/plan/AggregateNode.java

```diff
@@ -289,13 +310,18 @@ protected int getPartitions(KafkaTopicClient kafkaTopicClient) {
   private void collectAggregateArgExpressions(
       final List<Expression> expressions,
       final List<Pair<String, Expression>> aggArgExpansionList,
-      final Map<String, Integer> expressionNames
+      final Map<String, Integer> expressionNames,
+      final Map<String, String> expressionToInternalColumnNameMap
   ) {
     expressions.stream()
         .filter(e -> !expressionNames.containsKey(e.toString()))
         .forEach(expression -> {
-          expressionNames.put(expression.toString(), aggArgExpansionList.size());
-          aggArgExpansionList.add(new Pair<>(expression.toString(), expression));
+          String internalColumnName = "KSQL_INTERNAL_COL_" + aggArgExpansionList.size();
```

Comment:
Make this a package private static string variable, then use the variable in the tests as well.